web/profiles/custom/os2loop/modules/os2loop_messages/src/Helper/Helper.php

```diff
@@ -62,7 +62,7 @@ private function createMessage(EntityInterface $entity, string $operation) {
     if (NULL !== $template) {
       $message = Message::create(['template' => $template]);
       if ($entity instanceof NodeInterface) {
-        if ($entity->hasField('os2loop_notify_users') && FALSE == $entity->get('os2loop_notify_users')->getValue()[0]['value']) {
+        if ($entity->hasField('os2loop_notify_users') && FALSE == (bool) $entity->get('os2loop_notify_users')->getString()) {
```

Comment:
One final thing: `FALSE == (bool) $entity->get('os2loop_notify_users')->getString()` should be written as `!(bool) $entity->get('os2loop_notify_users')->getString()` (since `(bool) …` is a boolean value).